proto/spell.proto

```diff
@@ -52,4 +52,15 @@ enum SecondaryResourceType {
 	SecondaryResourceTypeBurningEmbers  = 108647;
 	SecondaryResourceTypeSoulShards     = 117198;
 	SecondaryResourceTypeHolyPower      = 138248;
-}
\ No newline at end of file
+}
+
+message RppmMod {
+    double coefficient = 1;
+    oneof mod_type {
+        int32 haste = 2;
+        int32 crit = 3;
```

Comment:
`bool` types might be clearer here since the value isn't actually used for anything.